docs/src/components/menu/v2menu.tsx

```diff
@@ -9,7 +9,16 @@ interface LinksProps {
 
 const Links: React.FunctionComponent<LinksProps> = (props) => (
   <ul className={props.listClass}>
-    <li className={props.itemClass}>
+    <li
+      className={props.itemClass}
+      onClick={() => {
+        window.gtag('event', 'Link out', {
+          event_category: 'Navigation',
+          event_label: 'Github Pages',
+          value: 0,
```

Comment:
any way to get info on what github page it is for value?

Reply:
Well it leads to the root page of of our autogenerated Github pages docs [here](https://open-olive.github.io/loop-development-kit/ldk/javascript/). So I thought this was appropriate, but I can change it to something else

Reply:
works for me! ha